tests/integration/run_tests.py

```diff
@@ -1109,8 +1141,8 @@ async def main():
             "description": "submission test folder 1",
         }
         submission_folder_id = await post_folder(sess, submission_folder)
-        await test_get_folders(sess, submission_folder_id)
-        await test_get_folders_objects(sess, submission_folder_id)
+        # await test_get_folders(sess, submission_folder_id)
```

Comment:
I assume this is still in progress, or any other reason these lines were commented ?

Reply:
Whoops. Used this a bypass before the existence of `clean_db.py` and forgot to undo the commenting out.